docs/csslint-options.md

```diff
@@ -86,6 +86,35 @@ For the current csslint version, the following formatters are supported:
 
 For an explanation of those formatters, [check the csslint wiki](https://github.com/stubbornella/csslint/wiki/Command-line-interface), section Options/--format.
 
+#### customFormatter
```

Comment:
Ah, good point. Done.

Then there's an option called `formatter` as well as `formatters`. Might be confusing?